doc/modules/linear_model.rst

```diff
@@ -1156,9 +1182,13 @@ Examples of use cases include:
 * Risk modeling / insurance policy pricing:  number of claim events /
   policyholder per year (Poisson), cost per event (Gamma), total cost per
   policyholder per year (Tweedie / Compound Poisson Gamma).
+* Credit Default / Fraud Detection: number of fraudulent transactions from a
+  region (Bernoulli)
 * Predictive maintenance: number of production interruption events per year
   (Poisson), duration of interruption (Gamma), total interruption time per year
   (Tweedie / Compound Poisson Gamma).
+* Medical Drug Testing: number of cured patients in a set of trials / number of
+  patients experiencing side effects (Bernoulli)
```

Comment:
Same comment here. Maybe you could also introduce a 3 class classification problem with mutually exclusive classes, e.g. sentiment analysis with positive, neutral and negative sentiments. Although, in this case, one could argue that this is actually original regression because of the ordering of the 3 classes. Maybe there is another simple example where the classes are not naturally ordered.

Reply:
One example that I could think of is that of a News Classification Problem where we are classifying news articles into 3 categories namely Business News, Politics and Entertainment news.